packages/frontend/src/components/Election/Admin/AddElectionRoll.tsx

```diff
@@ -38,7 +38,7 @@ const AddElectionRoll = ({ onClose }: { onClose: () => void }) => {
     const onSubmit = async (e) => {
         e.preventDefault()
         try {
-            const rows = voterIDList.split('\n')
+            const rows = voterIDList.split('\n').filter(row => row.trim().length > 0)
```

Comment:
This filter expression could also be written equivalently as `row => row.trim()` because trim will return the empty string which is falsey in ts/js